gui/src/components/commons/BaseModal.tsx

```diff
@@ -17,14 +17,17 @@ export function BaseModal({
       overlayClassName={
         props.overlayClassName ||
         classNames(
-          'fixed top-0 right-0 left-0 bottom-0 flex flex-col justify-center items-center w-full h-full bg-background-90 bg-opacity-60 z-20'
+          'fixed top-0 right-0 left-0 bottom-0 flex flex-col justify-center',
+          'items-center w-full h-full bg-background-90 bg-opacity-60 z-20'
         )
       }
       className={
         props.className ||
         classNames(
           props.className as string,
-          'items-center focus:ring-transparent focus:ring-offset-transparent focus:outline-transparent outline-none bg-background-60 p-6 rounded-lg text-white'
+          'items-center focus:ring-transparent focus:ring-offset-transparent',
```

Comment:
Please use multiline strings, not multiple, `classNames` has a tendency to not apply the 2nd string

Reply:
Wrong it does work with multiple string. Also Using multiline string create wierd class definition in the dom. It does not remove the \n for some reasons. this is why we are not using it. I havent checked in a while if it is still like that tho, so might be worth checking again.

Reply:
Also multiline string does not allow for dynamic classes without ending up with really ugly code or having a string with a bunch of empty spaces

Reply:
Okay, fair